.github/workflows/build-tarball.yml

```diff
@@ -26,8 +26,8 @@ jobs:
         run: |
           export DISTTYPE=nightly
           export DATESTRING=`date "+%Y-%m-%d"`
-          export COMMIT=xxxx
-          ./configure && make tar -j8
+          export COMMIT=$(git rev-parse --short "$GITHUB_SHA")
```

Comment:
Just out of curiousity, the `$(...)` bit only shortens `$GITHUB_SHA`, right?

Reply:
Yes. It makes the length of the commit hash consistent with the equivalent parameter for the release jobs in Jenkins:
![image](https://user-images.githubusercontent.com/5445507/88435671-9398bc80-cdfa-11ea-985a-af821b29957d.png)

Reply:
Ok, I see… if Jenkins always uses 10 characters, I think it might make sense to be explicit about that here:

```suggestion
          export COMMIT=$(git rev-parse --short=10 "$GITHUB_SHA")
```

but you can obviously also completely ignore this because it’s a very very minor suggestion :)